http/src/request/application/set_command_permissions.rs

```diff
@@ -140,17 +247,67 @@ mod tests {
         ))
     }
 
+    #[allow(unused)]
     #[test]
-    fn test_correct_validation() {
+    fn test_correct_validation() -> Result<(), Box<dyn Error>> {
```

Comment:
Imo, there is a weird mix of `.expect()` calls and `?` error handling in this test. Can it just stick to the usual unwrap-style to stay unified?

Reply:
There's an `expect` because it's used on an `Option`. Using `?` in a function that returns `Result` is still unstable. Additions for the past few months have preferred returning `Result`s in tests over unwrapping where possible, so IMO it should be left as is.

Reply:
Works for me.